### PR TITLE
Force ci to use fakes3 on its fixed port

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/elife-xpub/issues/1064#issuecomment-442754324

Exploring with the build why was [this commit](https://github.com/elifesciences/elife-xpub/pull/1117/commits/b07650cb6fc0a315218b44c9c47be64a7210f4cf) necessary. I am trying to:

- use `test` and `docker-compose.ci.yml` for all automated testing
- use `ci` and Kubernetes, Helm for review environments

`elife-xpub--ci` is dead so won't use any environment.

### CODE REVIEW

File: config/ci.js [hunk 1]
```diff
@@ -3,10 +3,7 @@ const AWS = require('aws-sdk')
 module.exports = {
   aws: {
     s3: {
-      endpoint: new AWS.Endpoint(
-        // randomise port to avoid conflicts in parallel test runs
-        `http://localhost:${Math.floor(Math.random() * 50000) + 15000}`,
-      ),
+      endpoint: new AWS.Endpoint('http://localhost:4569'),
```
Comment (damnhipster) [inline config/ci.js]: Is localhost:4569 mapped to the fakes3 pod in Kubernets?

Comment (giorgiosironi) [reply config/ci.js]: Yes, `localhost` cover the whole pod which hosts both `elife-xpub` and `fakes3`:
https://github.com/elifesciences/elife-xpub-formula/blob/master/helm/elife-xpub/templates/deployment.yaml#L73

Comment (giorgiosironi) [reply config/ci.js]: For a testing environment it wasn't worth it to have a separate Deployment and Service

**Comment by giorgiosironi** [comment]
Uploading a manuscript in the review environment works in this PR (http://35.172.178.78:31313/submit/471ded66-5c8b-42fe-866a-bc071aee1a72/submission), and http://35.172.178.78:31313/ping works too (although it took some seconds to start up).

**Comment by giorgiosironi** [comment]
@diversemix, who added this config?

**Comment by damnhipster** [comment]
> @diversemix, who added this config?

This was added as a fix for failing tests when adding the `S3_ENDPOINT` environment variable, but it seems that it wasn't necessary in the end.

**Comment by giorgiosironi** [comment]
Ok let's go

**Pull request merged**